Explorer/Assets/DCL/AvatarRendering/Loading/Components/ITrimmedAvatarAttachment.cs

```diff
@@ -0,0 +1,51 @@
+using CommunicationData.URLHelpers;
+using DCL.AvatarRendering.Loading.DTO;
+using DCL.AvatarRendering.Wearables;
+using ECS.StreamableLoading.Common.Components;
+using ECS.StreamableLoading.Textures;
+
+namespace DCL.AvatarRendering.Loading.Components
+{
+    public interface ITrimmedAvatarAttachment : IThumbnailAttachment
+    {
+        /// <summary>
+        ///     If null - promise has never been created, otherwise it could contain the result or be un-initialized
+        /// </summary>
+        StreamableLoadingResult<SpriteData>.WithFallback? ThumbnailAssetResult { get; set; }
+
+        TrimmedAvatarAttachmentDTO TrimmedDTO { get; }
+
+        public URLPath GetThumbnail() =>
+            new (TrimmedDTO.thumbnail);
+
+        public string ToString() =>
```

Comment:
Interfaces don't have a `ToString()` common method to override; here you just introduced a separate method which is never used